sql/movies.sql

```diff
@@ -0,0 +1,5 @@
+create table movies(movie_id int not null , 
+				   movie_name varchar(255) not null unique,
+				   release_date date not null,status boolean not null,
+					end_date date not null, primary key(movie_id)
```

Comment:
remove not null for end date,use serial for movie id column, status should contain dfault value true